e2e-polybft/e2e/jsonrpc_test.go

```diff
@@ -157,9 +157,26 @@ func TestE2E_JsonRPC(t *testing.T) {
 		require.NoError(t, txn.Wait())
 		require.True(t, txn.Succeed())
 
-		resp, err := client.GetStorageAt(txn.Receipt().ContractAddress, ethgo.Hash{}, ethgo.Latest)
+		target := txn.Receipt().ContractAddress
+
+		resp, err := client.GetStorageAt(target, ethgo.Hash{}, ethgo.Latest)
 		require.NoError(t, err)
 		require.Equal(t, "0x0000000000000000000000000000000000000000000000000000000000000000", resp.String())
+
+		setValueFn := abi.MustNewMethod("function setValue(uint256 _val) public")
+
+		newVal := big.NewInt(1)
+
+		input, err := setValueFn.Encode([]interface{}{newVal})
```

Comment:
send just `0x1`?

Reply:
Since this field is `uint256` type, we must use *big.Int (and not `int` or `uint64`). https://github.com/0xPolygon/polygon-edge/pull/1546/files#diff-3c7abd257acc39f7205b0b13f903fd280b1b38d3112ddadcf114a9ff72a3e9fdL5